package.json

```diff
@@ -58,7 +58,7 @@
     "istanbul": "^0.4.2",
     "lodash": "^4.13.1",
     "mapbox-gl-test-suite": "mapbox/mapbox-gl-test-suite#e78f09dce98080a9a6fa436d11fdf6fc2f271d7a",
-    "nyc": "^6.1.1",
+    "nyc": "6.4.0",
```

Comment:
This was about a different issue where one of nyc `6.5.0` subdeps was broken and erroring on Travis, not related to headless tests.